### ISSUE TITLE
TypeError: ofTypes.includes is not a function in Node v5.11.1

### ISSUE THREAD

**Issue by garetmckinley** [JSBros/hedron#43]
When integrating hedron into ViMD and testing on Travis-CI under node v5.11.1, I'm receiving the following error:

    TypeError: ofTypes.includes is not a function

      at node_modules/hedron/dist/hedron.js:7482:60
      at mapSingleChildIntoContext (node_modules/react/lib/ReactChildren.js:107:26)
      at traverseAllChildrenImpl (node_modules/react/lib/traverseAllChildren.js:77:5)
      at traverseAllChildrenImpl (node_modules/react/lib/traverseAllChildren.js:93:23)
      at traverseAllChildren (node_modules/react/lib/traverseAllChildren.js:172:10)
      at mapIntoWithKeyPrefixInternal (node_modules/react/lib/ReactChildren.js:127:3)
      at Object.mapChildren [as map] (node_modules/react/lib/ReactChildren.js:149:3)
      at passOn (node_modules/hedron/dist/hedron.js:7478:42)
      at RowContainer (node_modules/hedron/dist/hedron.js:7502:21)
      at node_modules/react-test-renderer/lib/ReactCompositeComponent.js:306:16

I'm guessing it requires a polyfill or something for it. I don't have time to investigate right now, so I'm putting this up to reference later 👌 

[Here is the Travis-CI build](https://travis-ci.org/JSBros/ViMD/jobs/182898976) that is causing the error.

**Comment by stephengfriend** [comment]
I ran into this issue on v4.3.2, as well. Is Hedron only compatible with `>= v6.x.x` ?

**Comment by garetmckinley** [comment]
Hmm, nice to know it wasn't an isolated incident. Going to ramp up the priority on this and investigate it this coming week. I'm on the road all weekend so unable to do much till monday, I'll keep this thread updated 👍

**Comment by stephengfriend** [comment]
I think it's just an issue with `Array.prototype.includes` not existing. I'm testing this out with an `array-includes` shim.

**Comment by stephengfriend** [comment]
I used `array-includes` package and added the following to my project entry...

```
delete Array.prototype.includes;
require('array-includes').shim();
```

This fixed it for me in, v4.3.2
